### ISSUE TITLE
Altmetrics badges show too close to the above row in simple/full item viw

### ISSUE THREAD

**Issue by AmberPoo1** [VTUL/vtechworks#206]
the Altmetrics badges show too close to the above row, we should add some space between two rows.

**Comment by alawvt** [comment]
@AmberPoo1, thanks for noticing this. The item that I tested for the Altmetrics badge didn't have a Creative Commons license, so I missed the conflict with the CC license box. I agree some space between them would be good.

**Comment by AmberPoo1** [comment]
Hi @amandafrench , I saw your commit of adding padding to Altmetrics badges, I tested on my local machine, it did work:
![screen shot 2016-02-26 at 1 37 19 pm](https://cloud.githubusercontent.com/assets/13734326/13361723/6169ae3e-dc8e-11e5-89d7-01a9587374db.png)

Maybe we didn't merge this change to dev machine?

Additionally, I make some improvement on Altmetrics badegs, including:
1. Get “data-handle” value from metadata “identifer.handle”.
2 Open a new page when click on Altmetrics badges.

The related pull request is  Improve #207 altmetrics badges. @alawvt, I have assigned this pull request to you. Thank you!
